src/main/resources/db/migration/V1.7__Add_RSS_Number.sql

```diff
@@ -0,0 +1,4 @@
+-- Add RSS number field to Beacon use, as it will be needed for fishing vessels
+
+ALTER TABLE beacon_use
+    ADD COLUMN rssNumber text;
```

Comment:
@Ninamma Column should be named lowercase underscore (can't remember the proper name for it 😆 )

So it should be `ADD COLUMN rss_number`